packages/flutter/test/painting/noop_codec.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+class NoopCodec implements Codec {
```

Comment:
It looks like `NoOpX` is a more common casing convention in the Flutter repo: https://github.com/search?q=repo%3Aflutter%2Fflutter%20Noop&type=code

```suggestion
class NoOpCodec implements Codec {
```

I don't feel strongly about this though